### PR TITLE
Better language for useless assignment with operator message

### PR DESCRIPTION
**Pull request by SherSpock**
Lightning PR I think would help others who have had trouble with the language of this cop.

All tests pass. The two "pending" failures are also present on master.

### CODE REVIEW

File: lib/rubocop/cop/lint/useless_assignment.rb [hunk 1]
```diff
@@ -89,7 +89,7 @@ def operator_assignment_message(scope, assignment)
           return unless assignment.meta_assignment_node
                                   .equal?(return_value_node)
 
-          " Use just operator `#{assignment.operator.sub(/=$/, '')}`."
+          " Use just operator `#{assignment.operator.sub(/=$/, '')}` instead of `#{assignment.operator}`."
```
Comment (bbatsov) [inline lib/rubocop/cop/lint/useless_assignment.rb]: I guess we can simplify this to " Use `x` instead of `y`.". Clear and concise.

Comment (SherSpock) [inline lib/rubocop/cop/lint/useless_assignment.rb]: I'm down with it and I made the change

**Comment by bbatsov** [comment]
The build is failing.

**Comment by SherSpock** [comment]
Interesting. Yeah, I see that.

Looking at the build I'm hoping it's just an RC lint failure from a line being too long.  

I'm taking a look at it now

**Comment by SherSpock** [comment]
Looks like that was it. We're green now.

My fault for not linting. RuboCop runs automatically every time we run our tests at work and I've gotten used to it.

**Pull request merged**